minerl/data/pipeline/publish.py

```diff
@@ -458,6 +458,14 @@ def publish(n_workers=56, parallel=True):
         print(open('errors.txt', 'r').read())
 
 
+def _make_tar(output_tar_path: str, folders: List[str]) -> None:
+    with tarfile.open(output_tar_path, "w") as archive:
+        logging.info('Generating archive {}'.format('data_texture_0_low_res.tar'))
```

Comment:
Likely this should be logging.info('Generating archive {}'.format(output_tar_path.split('/')[-1])